std/format/read.d

```diff
@@ -415,133 +526,40 @@ uint formattedRead(R, Char, S...)(auto ref R r, const(Char)[] fmt, auto ref S ar
 }
 
 /**
- * Reads a value from the given _input range according to spec
- * and returns it as type `T`.
- *
- * Params:
- *     T = the type to return
- *     input = the _input range to read from
- *     spec = the `FormatSpec` to use when reading from `input`
- * Returns:
- *     A value from `input` of type `T`
- * Throws:
- *     A `FormatException` if `spec` cannot read a type `T`
- * See_Also:
- *     $(REF parse, std, conv) and $(REF to, std, conv)
- */
-T unformatValue(T, Range, Char)(ref Range input, scope const ref FormatSpec!Char spec)
-{
-    return unformatValueImpl!T(input, spec);
-}
-
-/// Booleans
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
-
-    auto str = "false";
-    auto spec = singleSpec("%s");
-    assert(unformatValue!bool(str, spec) == false);
-
-    str = "1";
-    spec = singleSpec("%d");
-    assert(unformatValue!bool(str, spec));
-}
-
-/// Null values
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
+Reads a value from an _input range and converts it according to a
+format specifier.
 
-    auto str = "null";
-    auto spec = singleSpec("%s");
-    assert(str.unformatValue!(typeof(null))(spec) == null);
-}
-
-/// Integrals
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
-
-    auto str = "123";
-    auto spec = singleSpec("%s");
-    assert(str.unformatValue!int(spec) == 123);
-
-    str = "ABC";
-    spec = singleSpec("%X");
-    assert(str.unformatValue!int(spec) == 2748);
-
-    str = "11610";
-    spec = singleSpec("%o");
-    assert(str.unformatValue!int(spec) == 5000);
-}
-
-/// Floating point numbers
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
-    import std.math : isClose;
-
-    auto str = "123.456";
-    auto spec = singleSpec("%s");
-    assert(str.unformatValue!double(spec).isClose(123.456));
-}
-
-/// Character input ranges
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
-
-    auto str = "aaa";
-    auto spec = singleSpec("%s");
-    assert(str.unformatValue!char(spec) == 'a');
-
-    // Using a numerical format spec reads a Unicode value from a string
-    str = "65";
-    spec = singleSpec("%d");
-    assert(str.unformatValue!char(spec) == 'A');
+Params:
+    input = an $(REF_ALTTEXT input range, isInputRange, std, range, primitives),
+            where the formatted input is read from
+    spec = a $(MREF_ALTTEXT format string, std,format)
+    T = type to return
+    Range = the type of the input range `input`
+    Char = the character type used for `spec`
 
-    str = "41";
-    spec = singleSpec("%x");
-    assert(str.unformatValue!char(spec) == 'A');
+Returns:
+    The value read.
```

Comment:
I think that the old message (A value from `input` of type `T`) is more better and more descriptive